utils/wasi-nn/install-pytorch.sh

```diff
@@ -15,17 +15,17 @@ PYTORCH_SHA="b76d6dd4380e2233ce6f7654e672e13aae7c871231d223a4267ef018dcbfb616"
 
 for i in "$@"; do
   case $i in
-    --disable-cxx11-abi)
-      PYTORCH_LINK="libtorch"
-      PYTORCH_SHA="b5ddadc9addc054d8503f4086546f0cbcfdc3fc70087863bbd7b0e3300e3247f"
-      shift
-      ;;
+  --disable-cxx11-abi)
```

Comment:
I think this file is not necessary to be modified?

Reply:
Yeah... It seems like the styling tool of my editor automatically modify it. It didn't involve any functional change.